react/features/toolbox/components/native/OverflowMenu.js

```diff
@@ -86,6 +87,7 @@ class OverflowMenu extends Component<Props> {
                 }
                 <LiveStreamButton { ...buttonProps } />
                 <TileViewButton { ...buttonProps } />
+                <InviteButton { ...buttonProps } />
```

Comment:
What are these invite button changes for? Ignoring that they probably should be in another commit, like saul said, I'm curious why you had to make these changes while adding chat.

Reply:
I reused the spot of the invite button (leftmost side of the ToolBox) to place the chat button there, so I put the invite button to the ... menu.